simplecpp.cpp

```diff
@@ -1752,8 +1753,15 @@ static bool realFileName(const std::string &f, std::string *result)
 
     // Lookup filename or foldername on file system
     WIN32_FIND_DATAA FindFileData;
-    HANDLE hFind = FindFirstFileA(&buf[0], &FindFileData);
-    if (hFind == INVALID_HANDLE_VALUE)
+	HANDLE hFind = INVALID_HANDLE_VALUE;
+	// only fetch once
+	static BOOL bIsWindows7OrGreater = IsWindows7OrGreater();
```

Comment:
I don't want that this is static because that is not threadsafe. Is this a slow function or is it ok that it is called every time?

Reply:
I will benchmark FIND_FIRST_EX_LARGE_FETCH again and if its unnecessary, remove the whole check alltogether

Reply:
Move static to file scope to resolve the multi-threading issue!?